arches/__init__.py

```diff
@@ -6,6 +6,8 @@
 except ModuleNotFoundError as e:
     print(e)
 
+# Change this every time you release a new version
+# For example, for Keystone version 1.2.2 use VERSION = (1, 2, 2, "final", 0)
```

Comment:
Should we be setting this by environment variable perhaps?

Reply:
Setting via KEYSTONE-SOFTWARE-VERSION env var. Note changes in https://hedev.visualstudio.com/Inventory/_git/keystone-config?path=/set_environment_variables.bat